pages/docs/configuration/rag_api.mdx

```diff
@@ -64,10 +64,11 @@ Here are some notable configurations:
 - `CHUNK_OVERLAP`: The overlap between chunks during text processing. Default is "100".
 - `EMBEDDINGS_PROVIDER`: The embeddings provider to use. Options are "openai", "azure", "huggingface", "huggingfacetei", or "ollama". Default is "openai".
 - `EMBEDDINGS_MODEL`: The specific embeddings model to use from the configured provider. Default is dependent on the provider; for "openai", the model is "text-embedding-3-small".
+- `OLLAMA_BASE_URL`: It should be provided if RAG API runs in docker and usually is `http://host.docker.internal:11434`. 
 
 There are several more configuration options.
 
-For a complete list and their descriptions, please refer to the [RAG API repo.](https://github.com/danny-avila/rag_api/)
+For a complete list and their descriptions, please refer to the [RAG API repo.](https:\/\/github.com/danny-avila/rag_api/)
```

Comment:
I missed this accident changed in web interface. Please remove it. I cannot find this branch anymore 🙈 Sorry for the trouble

Reply:
no worries